packages/svelte/src/lib/containment/styles/surface-container.css

```
@@ -0,0 +1,29 @@
+@reference '$style/index.css';
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@reference" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)